lib/features/calendar/data/models/calendar_model.dart

```diff
@@ -72,6 +74,31 @@ class CalendarModel {
     );
   }
 
+  factory CalendarModel.fromSeribase(Map<String, dynamic>? data) {
```

Comment:
is the factory _fromSeribase_ being used from the repository or presentation class?

Reply:
repository